PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -22,7 +22,7 @@ public static void Main(string[] args)
             Console.WriteLine("You got " + GetPercentCorrect(numberCorrect, questions.Length) + " correct");
         }
 
-        public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
+        public static string GetPercentCorrect(double numberCorrectAnswers, double numberOfQuestions)
         {
             return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
```

Comment:
Not a bad solution, but would have also added a Math.Round in case I still had a decimal after my division.
```suggestion
return Math.Round((float)numberOfCorrectAnswers / numberOfQuestions * 100) + “%”
```